### PR TITLE
Added namespacing for extension

### PR DESCRIPTION
**Pull request by ahoiroman**
This adds proper namespacing for the extension

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -50,11 +50,16 @@
...
+        'sgerli/flick:' => '',
+    ],
+
     'events' => [
         'view.scripts' => function ($event, $scripts) use ($app) {
 
-			$scripts->register('uikit-grid', 'app/assets/uikit/js/components/grid.min.js', 'uikit');
-			$scripts->register('uikit-lightbox', 'app/assets/uikit/js/components/lightbox.min.js', 'uikit');
+			$scripts->register('uikit-grid', 'sgerli/flick:app/assets/uikit/js/components/grid.min.js', 'uikit');
+			$scripts->register('uikit-lightbox', 'sgerli/flick:app/assets/uikit/js/components/lightbox.min.js', 'uikit');
```
Comment (sGerli) [inline index.php]: This two (61 and 62) lines don't work with `sgerli/flick:` because they require files from Pagekit, not the extension.

**Review by sGerli — CHANGES_REQUESTED** [review]
Sorry for the delay, I was on vacation and I thought that I would have more time than I did, but now I'm back. From `index.php` lines 61 and 62 need changes and from `FlickController.php` lines 18 and 53 need `sgerli/` to be added.

File: index.php (hunk 1 above)
Comment (ahoiroman) [reply index.php]: Wouldn't it be better to include those like this:

`$scripts->register('some-stuff', 'sgerli/flick:app/assets/my.css', ['uikit', 'uikit-grid', 'uikit-lightbox']);`

?

Comment (sGerli) [reply index.php]: That's already in `views/index.php` line 7, and we need to register those dependencies.

Comment (sGerli) [reply index.php]: After a bit more testing I realized we can completely remove this two lines (61, 62) and it still loads them as dependencies, but I still have the sort problem.

Comment (sGerli) [reply index.php]: So now just waiting for rebase with lines 18 and 53 from `FlickController.php` fixed.

**Pull request merged**